specs-go/config.go

```diff
@@ -79,6 +85,8 @@ type Platform struct {
 	OS string `json:"os"`
 	// Arch is the architecture
 	Arch string `json:"arch"`
+	// OSVersion is the version of the operating system. (this field is platform dependent)
+	OSVersion string `json:"os.version,omitempty" platform:"windows"`
```

Comment:
How do you expect this field to be used?  The consensus for config settings that aren't supported by all versions of the Linux kernel (all the config settings?) seems to be:

> Just return kernel errors in a reasonable way.  You don't have to check for this sort of thing ahead of time.

See [here][1] and #418.

[1]: https://github.com/opencontainers/runtime-spec/pull/397#issuecomment-223062579